tests/core/ckeditor/manual/assetscachekeycdn.html

```diff
@@ -0,0 +1,114 @@
+<div id="editor"></div>
+<button disabled id="loadButton">Load external editor</button>
+<div id="loading_box" style="margin: 20px 0"></div>
+<div id="editor_cdn">
+	<p>A simple editor</p>
+</div>
+<div id="fake_observable_item"></div>
+
+<script>
+	( function () {
+		if ( bender.tools.env.mobile || ( CKEDITOR.env.ie && CKEDITOR.env.version < 11 ) ) {
+			return bender.ignore();
+		}
+
+		// 1. Load local editor by default with bender.
+		// 2. Remove existing instance and add script with editor from CDN.
+		// 3. Wait for the new editor to be loaded from CDN and initialize it.
+		// 4. Add icons in IE <11 from local.
+		var editorVersion = '',
+			path = getCKEditorCORSLink( window.location ),
+			loadButton = CKEDITOR.document.getById( 'loadButton' );
+
+		CKEDITOR.replace( 'editor', {
+			// Set fake element as observableParent due to removing existing instance of the editor.
+			observableParent: document.getElementById( 'fake_observable_item' )
+		} );
+
+		CKEDITOR.once( 'instanceReady', function() {
+			loadButton.$.disabled = false;
+			loadButton.once( 'click', loadExternalEditor );
+		} );
+
+		function loadExternalEditor() {
```

Comment:
As you mentioned (https://github.com/ckeditor/ckeditor4/pull/5035#issue-1101523489), there is a lot of common code with `previewcdn`. It could be extracted to some helper file, but the common way in bender with `exported` and `global` in comments don't work. Not sure if linking via `<script>` is a good idea in bender... WDYT?

Reply:
TBH I'd keep them separated and come back to the idea if there is yet another CDN test.